circle.yml

```diff
@@ -27,13 +27,7 @@ dependencies:
 
 test:
   override:
-    - node -v
-    - npm run lint
```

Comment:
The advantage of having those steps one by one in .yml file is nice wrapping.
If npm run lint fails then it takes less than a second to figure out.

However if we put everything in one shell script looking up the reason to fail will involve a lot of scrolling.
<img width="500" alt="screen shot 2016-10-19 at 12 04 22 pm" src="https://cloud.githubusercontent.com/assets/2121048/19516532/4ac7b6f6-95f4-11e6-893d-6baab02534ad.png">